### PR TITLE
Added correct ios app image for SUSI.AI

### LINKED ISSUE #319: Add and correct information for section "SUSI.AI iOS App" and add app image
The new section "SUSI.AI iOS App" needs correct info, links and app image. Please help to do it.

### PR DESCRIPTION
**Pull request by sahilsaha7773**
- [Preview link](https://sahilsaha7773.github.io/labs.fossasia.org/)
- Fixes #319 

**Changes done**

Added correct ios app image.

### CODE REVIEW

**Comment by abishekvashok** [comment]
Susi iso?

**Comment by sahilsaha7773** [comment]
Yes

**Comment by sahilsaha7773** [comment]
Sorry I am editing it

**Comment by sahilsaha7773** [comment]
Done!!

**Comment by sahilsaha7773** [comment]
I will edit the codes too

**Comment by sahilsaha7773** [comment]
Done!!

**Comment by sahilsaha7773** [comment]
Sorry for the mistake

**Comment by sahilsaha7773** [comment]
@abishekvashok

**Comment by sahilsaha7773** [comment]
please review

**Comment by sahilsaha7773** [comment]
will do it

**Comment by sahilsaha7773** [comment]
Done!!

**Comment by sahilsaha7773** [comment]
@abishekvashok please review

**Comment by sahilsaha7773** [comment]
Please merge

File: index.html [hunk 1]
```diff
@@ -456,7 +456,7 @@ <h1 class="text-white">SUSI.AI iOS</h1>
 
 				<div class="go-right side-image col-sm-4 col-md-6">
 					<div class="background-image-holder">
-						<img class="background-image" alt="FOSSASIA Wordpress Plugins" src="img/asksusi.png">
+						<img class="background-image" alt="SUSI.AI IOS " style="width="100%"; src="img/susi_ios.png">
```
Comment (jogendra) [inline index.html]: Should have used iOS

**Pull request merged**